docs/project.rst

```diff
@@ -36,34 +36,36 @@ in January 2019.
 Committers
 ----------
 
-- @mistercrunch (Maxime "Max" Beauchemin)
-- @r39132 (Siddharth "Sid" Anand)
-- @criccomini (Chris Riccomini)
-- @bolkedebruin (Bolke de Bruin)
+- @aijamalnk (Aizhamal Nurmamat kyzy)
+- @alexvanboxel (Alex Van Boxel)
 - @artwr (Arthur Wiedmer)
-- @jlowin (Jeremiah Lowin)
+- @ash (Ash Berlin-Taylor)
+- @basph (Bas Harenslak)
+- @bolkedebruin (Bolke de Bruin)
+- @criccomini (Chris Riccomini)
 - @aoen (Dan Davydov)
-- @msumit (Sumit Maheshwari)
-- @alexvanboxel (Alex Van Boxel)
-- @saguziel (Alex Guziel)
-- @joygao (Joy Gao)
+- @dimberman (Daniel Imberman)
+- @feluelle (Felix Uellendall)
 - @fokko (Fokko Driesprong)
-- @ash (Ash Berlin-Taylor)
-- @kaxilnaik (Kaxil Naik)
-- @feng-tao (Tao Feng)
 - @hiteshs (Hitesh Shah)
 - @jghoman (Jakob Homan)
-- @XD-DENG (Xiaodong Deng)
-- @dimberman (Daniel Imberman)
-- @potiuk (Jarek Potiuk)
-- @basph (Bas Harenslak)
+- @jlowin (Jeremiah Lowin)
 - @jmcarp (Joshua Carp)
-- @KevinYang21 (Kevin Yang)
+- @joygao (Joy Gao)
 - @mik-laj (Kamil Breguła)
-- @aijamalnk (Aizhamal Nurmamat kyzy)
+- @kaxilnaik (Kaxil Naik)
+- @KevinYang21 (Kevin Yang)
+- @mistercrunch (Maxime Beauchemin)
 - @milton0825 (Chao-Han Tsai)
-- @feluelle (Felix Uellendall)
-
+- @msumit (Sumit Maheshwari)
+- @pltardif (Patrick Leo Tardif)
+- @potiuk (Jarek Potiuk)
+- @saguziel (Alex Guziel)
```

Comment:
This should be after "@r39132 Siddharth Anand" for alphabetical order :)

Reply:
Ok, I will sort it A-Z. I wanted to keep the order consistent with this provided by Apache (I haven't figured the rule :D )

Reply:
Aah I thought you wanted to sort it 😄 Apache sorts it based on (Date, Names) the person becomes committer. When Airflow graduated into a Top Level Project, the PPMC became PMC at the same dates. Hence, they are sorted according to Names.

Reply:
> Ok, I will sort it A-Z. I wanted to keep the order consistent with this provided by Apache (I haven't figured the rule :D )

But the apache list lists apache handles and the project.rst lists github handles?!